src/formatDate.js

```diff
@@ -50,7 +50,27 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const objectDate = {};
+  const separatorToFormat = toFormat[toFormat.length - 1];
+  const separatorFromFormat = fromFormat[toFormat.length - 1];
```

Comment:
You are using `toFormat.length` for `separatorFromFormat`. It should be `fromFormat.length` to correctly find the separator in the `fromFormat` string.